Dockerfile

```diff
@@ -14,33 +14,31 @@ RUN apt-get update -qq && cat /tmp/apt-packages.txt | xargs apt-get --yes --forc
 ADD conf/npm-packages.txt /tmp/npm-packages.txt
 RUN cat /tmp/npm-packages.txt | xargs npm install -g
 
-ADD pip-requires.txt /tmp/requirements_requires.txt
-RUN pip install -r /tmp/requirements_requires.txt  
-
 ADD pip-freeze.txt /tmp/requirements.txt
 RUN pip install --upgrade pip
 RUN pip install -r /tmp/requirements.txt  
 RUN pip install coveralls
 
-WORKDIR /code
-
-ADD ./ /code/
-
 RUN mkdir -p /var/log/ureport
+RUN mkdir -p /var/log/uwsgi
+RUN touch /var/log/ureport/django.log
+RUN chown -R www-data:www-data /var/log/ureport /var/log/uwsgi
 
 ADD conf/entrypoint.sh /usr/local/bin/entrypoint.sh 
 RUN chmod +x /usr/local/bin/entrypoint.sh
 
-RUN mkdir -p /etc/supervisor/conf.d && mkdir -p /var/log/supervisor
```

Comment:
Yeah I changed it back - this comment hasn't disappeared 'cause I've added this line back in below.